app/write_square_test.go

```diff
@@ -198,19 +154,19 @@ func Test_finalizeLayout(t *testing.T) {
 			blobTxs: generateBlobTxsWithNamespaces(
 				t,
 				[]appns.Namespace{ns1, ns3, ns3, ns1, ns2, ns2},
-				[][]int{{100}, {1400, 1000, 200, 200}, {420}},
+				[][]int{{100}, {1400, 900, 200, 200}, {420}},
 			),
-			expectedIndexes: [][]uint32{{4}, {8, 12, 5, 6}, {7}},
+			expectedIndexes: [][]uint32{{4}, {8, 11, 5, 6}, {7}},
 		},
 		{
 			squareSize:      4,
 			nonreserveStart: 4,
 			blobTxs: generateBlobTxsWithNamespaces(
 				t,
 				[]appns.Namespace{ns1, ns3, ns3, ns1, ns2, ns2},
-				[][]int{{100}, {1000, 1400, 200, 200}, {420}},
+				[][]int{{100}, {900, 1400, 200, 200}, {420}},
```

Comment:
switched to 900 instead of 1000, because 1000 used to be 2 shares before the namespace increase